packages/blade/src/@types/globals.d.ts

```diff
@@ -2,7 +2,7 @@
 import 'styled-components';
 import 'styled-components/native';
 import type { Theme } from '../components/BladeProvider';
-// and extend them!
+import '@testing-library/jest-dom';
```

Comment:
adding to make the types globally available for test files, otherwise I've to include this in every test file else `expect().toBeInTheDocument()` kind of method gives TS errors `cannot call method on 'any'`